samples/benchmarks/coremark/src/main.c

```diff
@@ -143,9 +147,9 @@ int main(void)
 
 	while (true) {
 		k_sem_take(&start_coremark, K_FOREVER);
-		LOG_INF("CoreMark started!");
-		LOG_INF("CPU FREQ: %d Hz", SystemCoreClock);
```

Comment:
How did it work previously without the extern declaration?

Reply:
See comment above. I've also tested that after the change, it still works on nrf52840.